Makefile

```
@@ -87,25 +157,42 @@ run-custatevec-tests: custatevec-tests
 run-hip-tests: hip-tests
 	$(MAKE) -C tests/ run-hip-tests
 
-PYTESTS = $(shell find qsimcirq_tests/ -name '*_test.py')
+PYTESTS := $(wildcard qsimcirq_tests/*_test.py)
 
 .PHONY: run-py-tests
 run-py-tests: pybind
-	for exe in $(PYTESTS); do if ! python3 -m pytest $$exe; then exit 1; fi; done
-
-.PHONY: run-tests
-run-tests: $(TESTS)
+	for exe in $(PYTESTS); do
+	    if ! python3 -m pytest $(PYTESTFLAGS) $$exe; then
+		echo "Pytest $(PYTESTFLAGS) $$exe returned an error"
+	        exit 1;
+	    fi;
+	done
```

Thus far PYTESTS expand to a single test, but even with several tests this should work as a single pytest execution:

```suggestion
	python3 -m pytest $(PYTESTFLAGS) $(PYTESTS)
```